### ISSUE TITLE
Ruby beautifying not working (with Rubocop)

### ISSUE THREAD

**Issue by fifiteen82726** [Glavin001/atom-beautify#1652]
# Description

# Input Before Beautification

This is what the code looked like before:

```rb
def testmethod
p 123 
end
```

# Expected Output

The beautified code should have looked like this:

```rb
def testmethod
  p 123 
end
```

# Actual Output

The beautified code actually looked like this:

```rb
def testmethod
p 123 
end
```

# Steps to Reproduce

1. Add code to Atom editor
2. Run command `Atom Beautify: Beautify Editor`
3. This beautified code does not look right!

# Debug

Here is a link to the `debug.md` Gist: https://gist.github.com/fifiteen82726/47515eb01a407bf87f7c1e445cd8c38e

# Checklist

I have:
- [x] Tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [x] Reloaded (or restarted) Atom to ensure it is not a caching issue
- [x] Searched through existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
so I know this is not a duplicate issue
- [x] Filled out the Input, Expected, and Actual sections above or have edited/removed them in a way that fully describes the issue.
- [x] Generated debugging information by executing `Atom Beautify: Help Debug Editor` command in Atom and added link for `debug.md` Gist to this issue

**Comment by hathawad** [comment]
I see this as well

**Comment by Glavin001** [comment]
https://github.com/Glavin001/atom-beautify/pull/1685 has been merged. Await next release.

**Comment by Glavin001** [comment]
Published to v0.30.0
